shapely/ops.py

```diff
@@ -164,6 +164,16 @@ def triangulate(geom, tolerance=0.0, edges=False):
     return [g for g in collection.geoms]
 
 
+def constrained_triangulate(geom):
+    """Creates the constrained Delaunay triangulation and returns a list of polygons
```

Comment:
Should I just reference the docstring in `constructive`, like we're starting to do here? https://github.com/shapely/shapely/pull/1673

Reply:
I think that should be the way to go. I've opened https://github.com/shapely/shapely/issues/1688 to discuss this further.

Reply:
I don't think it is needed to add a version to `ops.py`. The existing functions in ops.py are (for now) kept for backwards compatibility, but other new functions have only been added as top-level functions (such as in `constructive.py`, which is then imported from there in the main `shapely.*` namespace)

Reply:
Thanks, I wasn't sure what purpose having this in `ops.py` had :+1: